apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -1,8 +1,61 @@
 import React from 'react';
 import style from './expanded_curriculum_catalog_card.module.scss';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import PropTypes from 'prop-types';
 
-const ExpandedCurriculumCatalogCard = () => {
-  return <div className={style.expandedCardContainer}>Expanded Card</div>;
+const ExpandedCurriculumCatalogCard = ({
+  courseDisplayName,
+  duration,
+  gradeRange,
+  subjectsAndTopics,
+  deviceCompatibility,
+  description,
+  professionalLearningProgram,
+  video,
+  publishedDate,
+  selfPacedPlCourseOfferingPath,
+}) => {
+  return (
+    <div>
+      <div className={style.expandedCardContainer}>
+        <h4>{courseDisplayName}</h4>
```

Comment:
Since we're trying to more unanimously use the new `Header` components, would you mind swapping out the `h4` and `p` tags with those please? Reference use: [here](https://github.com/code-dot-org/code-dot-org/pull/52616/files#diff-00e3281ed7840013a6efe24701175be81f44062d1d1edc0ba5033415cacce1dcR327-R340). You can set which tag they are semantically and separately which one they should be styled like!